### PR TITLE
Prac 05 code review request

### PR DESCRIPTION
**Pull request by jameswongsonegoro**
Check code structure
@austinliandro
@Andre-Jonathan

### CODE REVIEW

File: prac_05/wimbledon.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-CP1404 Practical
```
Comment (austinliandro) [inline prac_05/wimbledon.py:2]: When i try this code, it's all working completely without any errors.

**Review by austinliandro — COMMENTED** [review]
Good coding

File: prac_05/word_occurrences.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 """
 Word Occurrences
-Estimate: 15 minutes (14.38)
-Actual:
+Estimate: 15 minutes
+Actual: 20 minutes
 """
 
 word_count = {}
```
Comment (austinliandro) [inline prac_05/word_occurrences.py:7]: it's a good variable

File: prac_05/word_occurrences.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 """
 Word Occurrences
```
Comment (austinliandro) [inline prac_05/word_occurrences.py]: This is a good code

File: prac_05/wimbledon.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 """
-CP1404 Practicals
+CP1404 Practical
 Wimbledon data-reading, processing and displaying
```
Comment (austinliandro) [inline prac_05/wimbledon.py:3]: This code shows all the detail that the task need to do

File: prac_05/hex_colours.py [hunk 5]
```diff
@@ -1,5 +1,5 @@
 """
-CP1404/CP5632 Practical
+CP1404 Practicals
```
Comment (Andre-Jonathan) [inline prac_05/hex_colours.py:2]: This code doesn't show if the user input an invalid input other than "". Add an if statement to check if the user inputs the correct input.

File: prac_05/emails.py [hunk 6]
```diff
@@ -1,5 +1,5 @@
 """
-CP1404 Practical
+CP1404 Practicals
```
Comment (Andre-Jonathan) [inline prac_05/emails.py:2]: This code looks good

**Pull request merged**